packages/tools/devtools/devtools-browser-extension/e2e-tests/app/container.ts

```diff
@@ -69,6 +70,10 @@ class CollaborativeText extends DataObject {
 		return this._text;
 	}
 
+	public get initialObjects(): Record<string, IFluidLoadable> {
```

Comment:
This isn't really a pattern we use with our 1P API. Instead of baking this in to our data object, I would recommend just passing in something like the following to the devtools initialization function:
```typescript
{ collaborativeText: text }
```

Reply:
Yes. For the collaborative text input area, we can just pass the shared string, but we also want to show the container data in devtool `data` tag. This is the data object will be passed to register in devtool.

Reply:
I'm suggesting that we move the creation of this record to the site where we initialize the devtools. That would give us the same behavior, but would remove the concept of "initialObjects" from our data object, since it isn't really needed for anything else, and doesn't match our data object example conventions.

I don't think this is harmful as is, but I also don't think it's necessary.